catalog/core/catalog-core-api/src/main/java/ddf/catalog/operation/OperationTransaction.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.operation;
+
+import java.io.Serializable;
+import java.util.List;
+
+import ddf.catalog.data.Metacard;
+
+/**
+ * Interface describing an operation transaction for the catalog
```

Comment:
Then that's what the javadoc needs to state. The current javadoc doesn't tell me how or when I should use this thing.